opg-golang-alpine/tests/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+services:
+  basegen:
+    build: ..
+    image: registry.service.opg.digital/opg-golang-alpine
```

Comment:
explain to future devs why we have both build and image, and the context in which we should use either